server/game/ongoingEffects/OngoingEffectLibrary.ts

```diff
@@ -46,7 +46,7 @@ export = {
     //     OngoingEffectBuilder.card.static(EffectName.AttachmentRestrictTraitAmount, object),
     // attachmentTraitRestriction: (traits) => OngoingEffectBuilder.card.static(EffectName.AttachmentTraitRestriction, traits),
     // attachmentUniqueRestriction: () => OngoingEffectBuilder.card.static(EffectName.AttachmentUniqueRestriction),
-    // blank: (blankTraits = false) => OngoingEffectBuilder.card.static(EffectName.Blank, blankTraits),
+    blank: (blankTraits = false) => OngoingEffectBuilder.card.static(EffectName.Blank, blankTraits),
```

Comment:
1. Can you remove "blankTraits" for now, there's nothing in SWU that erases traits currently so it just adds confusion imo. Will be easy to add back in later if we need it.

2. Blanking for units will need to have some different behaviors to handle the possibility of being blanked and then gaining new abilities, so we need to separate it out from blanking events. Could we name this "blankEventCard" or something